cmd/api/main.go

```diff
@@ -136,9 +137,24 @@ func RebuildPackageInit(ctx context.Context) (*apiservice.RebuildPackageDeps, er
 	if err != nil {
 		return nil, errors.Wrap(err, "creating attestation uploader")
 	}
-	d.LocalMetadataStore = rebuild.NewFilesystemAssetStore(memfs.New())
-	d.RemoteMetadataStoreBuilder = func(ctx context.Context, id string) (rebuild.LocatableAssetStore, error) {
-		return rebuild.NewGCSStore(context.WithValue(ctx, rebuild.RunID, id), "gs://"+*metadataBucket)
+	if *debugStorage == "" {
```

Comment:
LGTM but would you mind removing the diff and submitting this change first? Feels like a simpler change overall.